### ISSUE TITLE
Raspberry Pi problem

### ISSUE THREAD

**Issue by craig42** [shyiko/jabba#105]
Hi, jabba doesn't work on Raspberry Pi (Raspbian armv7l )

```sh
curl -sL https://github.com/shyiko/jabba/raw/master/install.sh | bash && . ~/.jabba/jabba.sh

Installing v0.9.0...

/home/pi/.jabba/bin/jabba does not appear to be a valid binary.

Check your Internet connection / proxy settings and try again.

If the problem persists - please create a ticket at https://github.com/shyiko/jabba/issue.
```
When I read jabba binairie file : 
```sh
cat .jabba/bin/jabba
Not Found
```

**Comment by craig42** [comment]
For information : 
```sh 
pi@Raspberry:~ $ getconf LONG_BIT
32
pi@Raspberry:~ $ uname -m
armv7l
```

**Comment by shyiko** [comment]
Hi @craig42. Sorry about delayed response (I've been on vacation). Fixed in 63ddad09a31769fae5ef76d255e5a20b4382bb1c.
